thoth/solver/cli.py

```diff
@@ -67,6 +69,30 @@ def cli(ctx=None, verbose=0):
     _LOG.debug("Debug mode is on")
 
 
+def _limit_memory() -> None:
+    """Limit memory to cgroup limit if we're inside a container.
+
+    This turn OOM killer errors into python MemoryError exceptions.
+    """
+    for file in [
+        '/sys/fs/cgroup/memory/memory.soft_limit_in_bytes', #  cgroups v1
+        '/sys/fs/cgroup/memory/memory.limit_in_bytes',
+        '/sys/fs/cgroup/memory.high', #  cgroups v2
+        '/sys/fs/cgroup/memory.max',
+    ]:
+        try:
+            with open(file) as limit:
+                memory = limit.read()
+                if memory == "max": # memory.high might contain "max" -> indicates to use the memory.max value
+                    continue
+                value = int(limit.read())
```

Comment:
isn't this meant to be:
```suggestion
                value = int(memory)
```

Reply:
Yeah, spot on, thanks :+1: